src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -241,7 +234,13 @@ def get_attrs(self, item_list, user, **kwargs):
             if isinstance(item, AlertRule):
                 alert_rule = serialized_alert_rules.pop(0)
                 if "latestIncident" in self.expand:
-                    alert_rule["latestIncident"] = incident_map.get(item.incident_id)
+                    incident = (
```

Comment:
Was this change necessary? Seems unrelated to the user changes? 

It also seems a bit strange to make both this query and the one above that fills `incident_map`, but LMK if I'm missing something (I probably am)

Reply:
I could do it in 2 separate PRs if that makes more sense, but this one is a subclass of `AlertRuleSerializer` and since I noticed the `user` issue only when `latestIncident` was passed to `expand`, I wanted to clean up other instances of it. I had to rewrite the queries because there is no `incident_id` column on an `AlertRule` model.

Reply:
Makes sense to me 👍